lib/rubocop/cop/naming/rescued_exceptions_variable_name.rb

```diff
@@ -150,6 +153,13 @@ def message(node)
           preferred_name = preferred_name(offending_name)
           format(MSG, preferred: preferred_name, bad: offending_name)
         end
+
+        def check_shadowed_variable_name(node)
+          node.each_descendant do |n|
+            return true if n.children.first.to_s == cop_config.fetch('PreferredName', 'e')
```

Comment:
Is there a problem if it replaces `preferred_name`?

```suggestion
            return true if n.children.first.to_s == preferred_name(n)
```